index.js

```diff
@@ -1,366 +1,413 @@
+client.on("message", async function(message) {
+  if (message.author.bot) return;
+  if (!message.content.startsWith(constants.PREFIX)) return;
 
-	const commandBody = message.content.slice(constants.PREFIX.length);
-	const args = commandBody.split(' ');
-	const command = args.shift().toLowerCase();
-	let resp, selector;
+  const commandBody = message.content.slice(constants.PREFIX.length);
+  const args = commandBody.split(" ");
+  const command = args.shift().toLowerCase();
+  let resp, selector;
 
-	// Command handler
-	switch (command) {
-		case "autoassign":
-			message.reply("Autoassign method [not implemented yet]");
-			break;
-		case "fa":
-			if (args[0] === undefined) {
-				return
-			}
-			let faverb = args[0].toLowerCase();
-			if (["join", "leave"].includes(faverb)) {
-				let role = message.guild.roles.cache.find(r => r.name === "Free Agent");
-				let member = message.member;
-				if (faverb === "join") {
-					member.roles.add(role);
-				} else {
-					member.roles.remove(role);
-				}
-			}
-			break;
-		case "ff":
-			message.channel.send(constants.HELP_MSG)
-			break;
-		case "mh":
-			resp = "";
-			if (args[0] === undefined) {
-				return;
-			}
-			selector = args[0];
-			if (Object.keys(constants.TEAM_MAP).includes(selector)) {
-				let translated = constants.TEAM_MAP[selector];
-				let res = await getTournamentStageMatches("5fb86cf65caa4f5e4bc861fd");
-				resp += "Pulling match history for " + translated +" : \n" +
-					"##################################### \n";
-				resp += "W: " + res[translated][0].length + " L: " + 
-					res[translated][1].length + "\n";
-				for (let i = 0; i < res[translated][0].length; i++) {
-					resp += res[translated][0][i] + " **Win** \n";
-				}
-				resp += "------------------------------------- \n"
-				for (let i = 0; i < res[translated][1].length; i++) {
-					resp += res[translated][1][i] + " **Loss** \n";
-				}
-			} else {
-				resp = "Not a valid argument. Format is as follows: !mh [TEAM]. ";
-			}
-			message.channel.send(resp);
-			break;
-		case "pd":
-			if (args.length <= 1) {
-				return;
-			}
-			let T1 = args[0]
-			let T2 = args[1]
-			let MN = args[0] + 'VS' + args[1]
-			const browser = await puppeteer.launch({
-        headless: true,
-				'args' : [
-					'--no-sandbox',
-					'--disable-setuid-sandbox'
-				]
-			});
-			const page = await browser.newPage();
-			await page.goto('http://prodraft.leagueoflegends.com/');
-			await page.type('#blue_team_name', T1)
-			await page.type('#red_team_name', T2)
-			await page.type('#match_name', MN)
-			await page.click('#root > div > div > button');
-			await delay(1000);
-			
-			let blue = await page.$('#blue_team_name')
-			let red = await page.$('#red_team_name')
-			let match = await page.$('#match_name')
+  // Command handler
+  switch (command) {
+    case "autoassign":
+      message.reply("Autoassign method [not implemented yet]");
+      break;
+    case "fa":
```

Comment:
Out of curiosity... What does case "FA" mean? Or FF for that matter?

Reply:
"fa" is soon to be deprecated - it adds the "Free Agent" role to an individual on discord. "ff" is the equivalent of a "!help" command, changed to ff due to other bots in the server already responding to "!help".

Reply:
ohh